cloudpickle/cloudpickle.py

```diff
@@ -1218,6 +1229,19 @@ class id will also reuse this enum definition.
     return _lookup_class_or_track(class_tracker_id, enum_class)
 
 
+def _get_parent(module):
+    """(Try to) access the parent module (or package) of a submodule"""
+    parent_name, _, module_name = module.__name__.rpartition('.')
+    if parent_name:  # pragma: no cover
```

Comment:
Why not coverage here? You added tests for this in this PR.

Reply:
Indeed, this is a leftover pragma that predated this PR. I'll remove it.